_posts/2018-09-19-release-1.5.4.md

```diff
@@ -0,0 +1,94 @@
+
+List of resolved issues:
+
+<h2>        Bug
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10011'>FLINK-10011</a>] -         Old job resurrected during HA failover
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10101'>FLINK-10101</a>] -         Mesos web ui url is missing.
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10115'>FLINK-10115</a>] -         Content-length limit is also applied to FileUploads
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10116'>FLINK-10116</a>] -         createComparator fails on case class with Unit type fields prior to the join-key
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10141'>FLINK-10141</a>] -         Reduce lock contention introduced with 1.5
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10142'>FLINK-10142</a>] -         Reduce synchronization overhead for credit notifications
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10150'>FLINK-10150</a>] -         Chained batch operators interfere with each other other
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10172'>FLINK-10172</a>] -         Inconsistentcy in ExpressionParser and ExpressionDsl for order by asc/desc
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10193'>FLINK-10193</a>] -         Default RPC timeout is used when triggering savepoint via JobMasterGateway
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10204'>FLINK-10204</a>] -         StreamElementSerializer#copy broken for LatencyMarkers 
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10255'>FLINK-10255</a>] -         Standby Dispatcher locks submitted JobGraphs
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10261'>FLINK-10261</a>] -         INSERT INTO does not work with ORDER BY clause
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10267'>FLINK-10267</a>] -         [State] Fix arbitrary iterator access on RocksDBMapIterator
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10293'>FLINK-10293</a>] -         RemoteStreamEnvironment does not forward port to RestClusterClient
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10314'>FLINK-10314</a>] -         Blocking calls in Execution Graph creation bring down cluster
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10328'>FLINK-10328</a>] -         Stopping the ZooKeeperSubmittedJobGraphStore should release all currently held locks
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10329'>FLINK-10329</a>] -         Fail with exception if job cannot be removed by ZooKeeperSubmittedJobGraphStore#removeJobGraph
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10330'>FLINK-10330</a>] -         Code Compilation failed in local mode
```

Comment:
@tillrohrmann This issue has not been fixed yet. I removed the fixed version in JIRA. Please update this entry here.

Reply:
Updated the release notes.